theme/js/toc.js

```diff
@@ -74,5 +74,35 @@
     }
 
     render[settings.showEffect]();
+
+    var tocLinkItems = $('#current > a.toc__link-item'); // Get all the TOC header links
```

Comment:
I would consider putting this in a different JS file because it's a very different function than the link creation, which is what the rest of this file is doing.

Reply:
That would also potentially allow us to progressively start shedding jQuery where we can.